index-flow.yml

```diff
@@ -22,13 +22,13 @@ executors:
       - executors.py
     needs: frame_extractor
   - name: image_encoder
-    uses: jinahub+docker://AudioCLIPImageEncoder/v0.6
+    uses: jinahub://AudioCLIPImageEncoder/v0.6
```

Comment:
it is better to use the docker version or else the users have to install quite a few dependencies